cypress/e2e/tests/migration/issues/single-issue/filter.test.ts

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    login,
+    getRandomApplicationData,
+    getRandomAnalysisData,
+    deleteByList,
+    clearAllFilters,
+} from "../../../../../utils/utils";
+import { Analysis } from "../../../../models/migration/applicationinventory/analysis";
+import { SEC, filterIssue } from "../../../../types/constants";
+let applicationsList: Array<Analysis> = [];
```

Comment:
**chore:** There is no need to create an array and to use `deleteByList` as there is just one application, you can do `application.delete()`

Reply:
Later I will add more applications here, so it is reasonable, don't worry. Tests will iterate between application's list.